docs/tutorials/projects/bedtime-songs-bot.md

```diff
@@ -5,7 +5,7 @@ weight: 60
 type: "docs"
 tags: ["mac", "app", "board", "webcam", "camera", "ml", "machine learning", "babysitter"]
 description: "Create a robot babysitter with a webcam and machine learning."
-imageAlt: "Image of Tess and their computer."
+images: [ "assets/tutorials/bedtime-songs-bot/robot_babysitter.gif" ]
```

Comment:
```suggestion
images: [ "/tutorials/bedtime-songs-bot/robot_babysitter.gif" ]
```
and it unfortunately needs to go in the static folder. Only images go into the assets folder - hugo does some resizing on them and then puts them into a folder in the deployed page. But it only does that for images